src/nupic/proto/SpatialPoolerProto.capnp

```diff
@@ -26,7 +26,7 @@ struct SpatialPoolerProto {
   minPctOverlapDutyCycles @15 :Float32;
   minPctActiveDutyCycles @16 :Float32;
   dutyCyclePeriod @17 :UInt16;
-  maxBoost @18 :Float32;
+  boostStrength @18 :Float32;
```

Comment:
@scottpurdy if we're not going to worry about handling old serialized SPs, should we still keep and deprecate the "maxBoost" in the proto? i.e. is it cool that Yuwei is reusing 18?